getCli.sh

```diff
@@ -3,26 +3,34 @@
 CLI_OS="na"
 CLI_UNAME="na"
 
+if [ $# -eq 0 ]
+  then
+	VERSION=\$latest
+	echo "Using latest version of jfrog cli"
```

Comment:
I suggest slightly modifying the message to:
**echo "Downloading the latest version of JFrog CLI..."**